src/config/steps/fing.coffee

```diff
@@ -1,8 +1,8 @@
 module.exports = {
-  name: 'fing',
-  view : 'steps/fing'
-  isActive: (instance) ->
-    instance.attributes \
-      && instance.attributes.context \
-      && instance.attributes.context.toLowerCase() is 'fing'
+    name: 'fing',
+    view : 'steps/fing'
+    isActive: (instance) ->
+        instance.attributes \
```

Comment:
The two following lines should also be indented, as they are the continuity of the first one, like this :
```js
    ...
    isActive: (instance) ->
        instance.attributes \
            && instance.attributes.context \
            && instance.attributes.context.tolowerCase() is 'fing'
```